### DISCUSSION TITLE
NDS icon from default theme

### DISCUSSION THREAD

**Discussion by light-and-ray** [OnionUI/Onion#1284] [General]
Hello do you have nds logo in default theme style? I want to change drastic icon to it, if you have it, could you give it to me, please

**Comment by Aemiii91** [comment]
https://github.com/OnionUI/Onion/blob/FEAT-NDS-integration/static/build/Icons/Default/nds.png

**Reply by light-and-ray** [reply]
Thanks 😍 
![MainUI_023](https://github.com/OnionUI/Onion/assets/33491867/a6788321-ee5d-43a6-b9c3-3163a2e1075b)

Glad to know than native drastic support is in development
